e2e_tests/integration/data-export.spec.ts

```diff
@@ -76,4 +76,11 @@ describe('Data export', () => {
       })
     })
   })
+
+  it('can export history', () => {
+    cy.executeCommand(':clear')
+    cy.executeCommand(':history')
+    cy.get('[data-testid="exportGrassButton"]').should('not.exist')
+    cy.get('[data-testid="exportHistoryButton"]').should('not.exist')
```

Comment:
hmm, this tests looks like it needs fixing?

Reply:
yes good catch